docs/compass/08-01-basic-flow-tutorial.md

```diff
@@ -99,18 +90,18 @@ For simplicity reasons, use the available Order Service as the sample external A
 
 Clean up your cluster after going through this tutorial. To do so, delete your resources in the following order:
 
-1. Go to the **Lambdas** tab, unfold the vertical option menu and delete your lambda.
+1. Go to the **Functions** tab and delete your Function.
 
-2. Go to the **Services** tab and delete `order-service`.
+2. Go to the **Services** tab and delete your services.
 
-3. Go to the **Deployments** tab and delete the `order-service` deployment.
+3. Go to the **Deployments** tab and delete your deployments.
 
-4. Go to the **APIs** tab and delete the `order-service` API.
+4. Go to the **API Rules** tab and delete your APIRules.
 
-5. Go to the **Instances** tab, navigate to **Services**, and deprovision your instance by selecting the trash bin icon.
+5. Go to the **Instances** tab and delete your instances.
 
-6. Go to the **Overview** section and unbind `test-app` from your Namespace.
+6. Go to the **Overview** section and **Unbind** `test-app` from your Namespace.
```

Comment:
```suggestion
6. Go to the **Overview** section and unbind `test-app` from your Namespace.
```
We do format UI elements (buttons) as code, but the `unbind` here does not explicitly refer to a UI element, it just names an action to perform. I would either rephrase this (see below), or remove the formatting (as per the suggestion). 

Rephrase:
```
6. Go to the **Overview** section and click **Unbind** to unbind `test-app` from your Namespace.
```